Sources/Purchasing/Package.swift

```diff
@@ -14,6 +14,39 @@
 
 import Foundation
 
+///
+/// Stores information about how a ``Package`` was presented.
+///
+@objc(RCPresentedOfferingContext) public final class PresentedOfferingContext: NSObject {
```

Comment:
Can you add this to the Swift/Obj-C API testers?

Reply:
OMG, how did I forget...